OpenRA.Mods.Common/Traits/Player/ProductionQueue.cs

```diff
@@ -47,8 +47,8 @@ public class ProductionQueueInfo : ITraitInfo
 		[Desc("Maximum number of items that can be queued across all actor types (0 = infinite).")]
 		public readonly int QueueLimit = 0;
 
-		[Desc("The build time is multiplied with this value on low power.")]
-		public readonly int LowPowerSlowdown = 3;
+		[Desc("The build time is multiplied with this percentage on low power.")]
+		public readonly int LowPowerModifier = 300;
```

Comment:
It may make sense to take this chance to reset the default to 100?  a 3x slowdown is quite arbitrary, and specific to the default C&C mods.